src/components/Groups/GroupsPage.tsx

```diff
@@ -147,7 +147,7 @@ const render = props => {
                   initialSortField={RESOURCE_MAP.users.initialSortField}
                   selectedItemId={userId}
                   Component={resource.ListItem}
-                  getData={resource.getList}
+                  getData={() => resource.getList({ groupsId: groupId })}
```

Comment:
Is getData is called with other options? maybe those should be spread in the getList call.

Reply:
Good call

Reply:
updated https://github.com/kids-first/kf-drp-auth-ui/pull/72/commits/ee9a626a89d897f199b3cd391039757365650c1d